tests/virt_autotest/install_package.pm

```diff
@@ -54,41 +55,19 @@ sub install_package {
     }
 
     if ($dependency_repo) {
-        if (check_var('ARCH', 's390x')) {
-            lpar_cmd("zypper --non-interactive --no-gpg-check ar -f ${dependency_repo} dependency_repo");
-            lpar_cmd("zypper --non-interactive --gpg-auto-import-keys ref");
-            lpar_cmd("zypper --non-interactive in $dependency_rpms");
-            lpar_cmd("zypper --non-interactive rr dependency_repo");
-        }
-        else {
-            zypper_call("--no-gpg-check ar -f ${dependency_repo} dependency_repo");
-            zypper_call("--gpg-auto-import-keys ref", 180);
-            zypper_call("in $dependency_rpms");
-            zypper_call("rr dependency_repo");
-        }
+        zypper_wrapper("--no-gpg-check ar -f $dependency_repo dependency_repo");
+        zypper_wrapper("--gpg-auto-import-keys ref");
+        zypper_wrapper("in $dependency_rpms");
+        zypper_wrapper("rr dependency_repo");
     }
 
-    ###SLE-12-SP4 arm64 installation has no KVM role selection
+    ### SLE-12-SP4 arm64 installation has no KVM role selection
     if (($repo_0_to_install =~ /SLE-12-SP4/m) && check_var('ARCH', 'aarch64')) {
         zypper_call("--gpg-auto-import-keys ref",         180);
         zypper_call("in -t pattern kvm_server kvm_tools", 300);
     }
 
-    #install qa_lib_virtauto
-    if (check_var('ARCH', 's390x')) {
-        lpar_cmd("zypper --non-interactive --gpg-auto-import-keys ref");
-        my $pkg_lib_data = "qa_lib_virtauto-data";
-        my $cmd          = "rpm -q $pkg_lib_data";
-        my $ret          = console('svirt')->run_cmd($cmd);
-        if ($ret == 0) {
-            lpar_cmd("zypper --non-interactive rm $pkg_lib_data");
-        }
-        lpar_cmd("zypper --non-interactive in qa_lib_virtauto");
-    }
-    else {
-        zypper_call("--gpg-auto-import-keys ref", 180);
-        zypper_call("in qa_lib_virtauto",         1800);
-    }
+    zypper_wrapper("in qa_lib_virtauto");
 
```

Comment:
@guoxuguang Please help review whether this will cause trouble for s390 without rm pkg first.

Reply:
1st, There should be performed a fresh host installation before trigger install_package, however, there was not stable BACKEND to support s390x lpar to do a fresh host installation on osd now. 
So, there was had to add check with qa_lib_virtauto-data pkg before install qa_lib_virtauto pkg on s390x lpar to clean up the testing environment on installed sles s390x lpar.
Please keep the above existed pkg steps on s390x lpar.
If not, there will be the incompeleted issue for s390x guest installation test run on osd.

Reply:
Refer to the http://openqa.nue.suse.com/tests/3048921/file/autoinst-log.txt sles12sp5 s390x guest installation, there was missed step about check qa_lib_virtauto-data pkg before install qa_lib_virtauto pkg. 
It is the risk for the existed s390x guest installation test runs on osd.